sdk/lib/crt/misc/i10output.c

```diff
@@ -24,22 +159,31 @@ struct _I10_OUTPUT_DATA {
  *      Native sets last byte of data->str to '0' or '9', I don't know what
  *      it means. Current implementation sets it always to '0'.
  */
-int CDECL MSVCRT_I10_OUTPUT(_LDOUBLE ld80, int prec, int flag, struct _I10_OUTPUT_DATA *data)
+int CDECL I10_OUTPUT(MSVCRT__LDOUBLE ld80, int prec, int flag, struct _I10_OUTPUT_DATA *data)
```

Comment:
Why the change of the function name? is it to differentiate it with Wine's implementation?

Reply:
Wine is getting rid of these MSVCRT_ prefixes. This is simply what it looks like in current wine head.